src/main/java/auto_scheduler/StudentScheduleRepository.java

```diff
@@ -6,6 +6,6 @@
 
 public interface StudentScheduleRepository {
 	void delete(String studentId)  throws FileNotFoundException,  IOException;
-	Collection<String> get(String studentId) throws FileNotFoundException,  IOException;
-	void write(String studentId,Collection<String> sectionNumbers) throws FileNotFoundException,  IOException;
+	Collection<Integer> get(String studentId) throws FileNotFoundException,  IOException;
+	void write(String studentId,Collection<Integer> sectionNumbers) throws FileNotFoundException,  IOException;
```

Comment:
@brominger why is this pull request changingthe sections numbers to integers?
what do we gain from this?

plus an int would not support a section number like `000X`